tests/processes/ProcessingTest.py

```diff
@@ -602,6 +603,71 @@ def check_prerequisites(cls):
                                   console_printer=self.console_printer)
         self.assertGreater(len(cache.data), 0)
 
+    def test_global_instantiation(self):
+        class TestThreeBear(Bear):
+
+            def __init__(self,
+                         file_dict,
+                         section,
+                         queue, timeout=0.1,
+                         debugger=False):
+                raise RuntimeError
+                Bear.__init__(self, section, queue, timeout, debugger)
+
+            def run(self, filename, file):
+                pass
+
+        class TestTwoBear(Bear):
+
+            BEAR_DEPS = {TestThreeBear}
+
+            def __init__(self,
+                         file_dict,
+                         section,
+                         queue, timeout=0.1,
+                         debugger=False):
+                raise RuntimeError
+                Bear.__init__(self, section, queue, timeout, debugger)
+
+            def run(self, filename, file):
+                pass
```

Comment:
If you don't run the bear, you don't have to override `run`. It raises by default `NotImplemented`, but that won't happen anyway :+1: